content/en/learn.md

```diff
@@ -17,62 +15,62 @@ There is tons of information about NumPy out there. If you are new to NumPy, we'
+If you have the basic know-how of NumPy and how to use it, you can try these advanced tutorials, books and videos for a better understanding of the specific NumPy concepts such as advanced indexing, splitting, stacking, linear algebra and more.
 
 <i class="fad fa-chalkboard"></i> **Tutorials**
 
-* [100 NumPy Exercises](http://www.labri.fr/perso/nrougier/teaching/numpy.100/index.html), Nicolas P. Rougier, 2016.
-* [An Introduction to NumPy and Scipy](https://engineering.ucsb.edu/~shell/che210d/numpy.pdf), M. Scott Shell, 2014. 
-* [Numpy Medkits](http://mentat.za.net/numpy/numpy_advanced_slides/), Stéfan van der Walt, 2008.
+* [100 NumPy Exercises](http://www.labri.fr/perso/nrougier/teaching/numpy.100/index.html) *by Nicolas P. Rougier*
+* [An Introduction to NumPy and Scipy](https://engineering.ucsb.edu/~shell/che210d/numpy.pdf) *by M. Scott Shell* 
+* [Numpy Medkits](http://mentat.za.net/numpy/numpy_advanced_slides/) *by Stéfan van der Walt*
 * [NumPy in Python (Advanced)](https://www.geeksforgeeks.org/numpy-python-set-2-advanced/)
 * [Advanced Indexing](https://www.tutorialspoint.com/numpy/numpy_advanced_indexing.htm)
 * [Machine Learning and Data Analytics with NumPy](https://www.machinelearningplus.com/python/numpy-tutorial-python-part2/)
 
 <i class="fas fa-books"></i> **Books**
 
-* [Python Data Science Handbook](https://www.amazon.com/Python-Data-Science-Handbook-Essential/dp/1491912057) by Jake Vanderplas
-* [Python for Data Analysis](https://www.amazon.com/Python-Data-Analysis-Wrangling-IPython/dp/1491957662) by Wes McKinney
-* [Numerical Python: Scientific Computing and Data Science Applications with Numpy, SciPy and Matplotlib](https://www.amazon.com/Numerical-Python-Scientific-Applications-Matplotlib/dp/1484242459) by Robert Johansson
+* [Python Data Science Handbook](https://www.amazon.com/Python-Data-Science-Handbook-Essential/dp/1491912057) *by Jake Vanderplas* 
+* [Python for Data Analysis](https://www.amazon.com/Python-Data-Analysis-Wrangling-IPython/dp/1491957662) *by Wes McKinney*
+* [Numerical Python: Scientific Computing and Data Science Applications with Numpy, SciPy and Matplotlib](https://www.amazon.com/Numerical-Python-Scientific-Applications-Matplotlib/dp/1484242459) *by Robert Johansson*
 
 <i class="far fa-file-video"></i> **Videos**
 
-* [Scientific Computing with Python - SciPy Japan, 2019](https://www.youtube.com/watch?v=cYugp9IN1-Q)
-* [Advanced Indexing operations in NumPy arrays](https://www.youtube.com/watch?v=2WTDrSkQBng) 
+* [Scientific Computing with Python](https://www.youtube.com/watch?v=cYugp9IN1-Q)
+* [Advanced Indexing Operations in NumPy Arrays](https://www.youtube.com/watch?v=2WTDrSkQBng) 
 
 ***
 
 ## NumPy Talks
 
-* [The Future of NumPy Indexing](https://www.youtube.com/watch?v=o0EacbIbf58) by Jaime Fernández (2016)
```

Comment:
The style changes are an improvement, but I'd like to keep the years because that gives relevant context. For talks about the state of NumPy or future plans, it matters when the talk was given.

Reply:
I see your point. I was going back and forth on whether to include the years for the talks.